charts/csm-application-mobility/values.yaml

```diff
@@ -50,32 +62,36 @@ velero:
 # Based on the objectstore being used , the velero plugin and its configuration may need to change! Below is the sample configuration for using aws object store.
 # GCP and Azure plugins configuration are different. See more details at: https://github.com/vmware-tanzu/helm-charts/blob/main/charts/velero/README.md
   configuration:
-    provider: aws
     backupStorageLocation:
-      name: default
-      bucket: velero-bucket
-      config: {}
-      #  region:
-      #  s3ForcePathStyle:
-      #  s3Url:
-      #  kmsKeyId:
-      #  resourceGroup:
-      #  The ID of the subscription containing the storage account, if different from the cluster’s subscription. (Azure only)
-      #  subscriptionId:
-      #  storageAccount:
-      #  publicUrl:
-      #  Name of the GCP service account to use for this backup storage location. Specify the
-      #  service account here if you want to use workload identity instead of providing the key file.(GCP only)
-      #  serviceAccount:
+      - name: default
+        bucket: my-bucket
+        provider: aws
+        config: {
+        # region:
+        # s3ForcePathStyle:
+        # s3Url:
+        # kmsKeyId:
+        # resourceGroup:
+        # The ID of the subscription containing the storage account, if different from the cluster’s subscription. (Azure only)
+        # subscriptionId:
+        # storageAccount:
+        # publicUrl:
+        # Name of the GCP service account to use for this backup storage location. Specify the
+        # service account here if you want to use workload identity instead of providing the key file.(GCP only)
+        # serviceAccount:
+        }
+    volumeSnapshotLocation:
+    - name: aws-default
+      provider: aws
 
   initContainers:
   - name: dell-custom-velero-plugin
-    image: dellemc/csm-application-mobility-velero-plugin:v0.3.0
+    image: dellemc/csm-application-mobility-velero-plugin:latest
```

Comment:
Similar to the application-mobility image, this should point to a version instead of latest. Check with a PO to see if this can be set to v1.0.0.